model/flow/ledger.go

```diff
@@ -16,6 +17,13 @@ func (r *RegisterID) String() string {
 	return fmt.Sprintf("%x/%x/%x", r.Owner, r.Controller, r.Key)
 }
 
+// Bytes returns a bytes representation of the RegisterID.
+//
+// the encoding uses the injective fingerprint module.
```

Comment:
```suggestion
// The encoding uses the injective fingerprint module.
```

Reply:
I believe a blank line in [Godoc](https://blog.golang.org/godoc) is used as a paragraph separation. It usually separate the high-level function description from the details. 

> Subsequent lines of text are considered part of the same paragraph; you must leave a blank line to separate paragraphs.